junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/TestInstantiationAwareExtension.java

```diff
@@ -65,7 +64,7 @@ public interface TestInstantiationAwareExtension extends Extension {
 	 * <li>{@link ExtensionContext#getTestMethod() getTestMethod()} is no longer
 	 * empty, unless the {@link TestInstance.Lifecycle#PER_CLASS PER_CLASS}
 	 * lifecycle is used.</li>
-	 * <li>If the callback adds a new {@link CloseableResource} to the
+	 * <li>If the callback adds a new {@link Store.CloseableResource} to the
```

Comment:
Please also mention `AutoCloseable` here as in the other places you've adapted the Javadoc.